tests/install.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
Can you add instructions in the readme? It tackles https://github.com/johmsalas/text-case.nvim/issues/74